### PR TITLE
Updating the getting started code.

### PR DESCRIPTION
**Pull request by sfshaza2**
Needed for the updated getting started codelabs.

### CODE REVIEW

File: startup_namer/step3_stateful_widget/lib/main.dart [hunk 1]
```diff
@@ -34,10 +34,10 @@ class RandomWordsState extends State<RandomWords> {
...
   // #docregion RWS-class-only
 }
-// #enddocregion RandomWordsState, RWS-class-only
+// #enddocregion _RandomWordsState, RWS-class-only
 
 // #docregion RandomWords
 class RandomWords extends StatefulWidget {
   @override
-  RandomWordsState createState() => RandomWordsState();
+  State<RandomWords> createState() => _RandomWordsState();
```
Comment (johnpryan) [inline startup_namer/step3_stateful_widget/lib/main.dart:42]: I think the return type for all of these `createState()` methods can be `_RandomWordsState` too, but `State<RandomWords>` is fine

**Comment by domesticmouse** [comment]
Hi @anthonymoretti,

Please raise a new issue on [flutter/flutter's issue tracker](https://github.com/flutter/flutter/issues) and add a reference to me (@domesticmouse) in the body of your issue and I'll make sure it is appropriately triaged and assigned. 

Thanks!

**Comment by kwalrath** [comment]
I believe codelab issues should be on [flutter/website's issue tracker](https://github.com/flutter/website/issues).

You can see all the open codelab issues here: https://github.com/flutter/website/issues?q=is%3Aissue+is%3Aopen+codelab+label%3Acodelab

**Comment by anthonymoretti** [comment]
Ok, thanks guys!

**Comment by kwalrath** [comment]
https://github.com/flutter/website/issues/4256

**Pull request merged**